src/Apps/Order/Components/TransactionSummary.tsx

```diff
@@ -16,77 +16,111 @@ export interface TransactionSummaryProps extends FlexProps {
   order: TransactionSummary_order
 }
 
-export const TransactionSummary: React.SFC<TransactionSummaryProps> = ({
-  order: {
-    itemsTotal,
-    taxTotal,
-    shippingTotal,
-    buyerTotal,
-    lineItems,
-    seller: { name },
-  },
-  ...others
-}) => {
-  const {
-    artist_names,
-    title,
-    date,
-    shippingOrigin,
-    image: {
-      resized_transactionSummary: { url: imageURL },
-    },
-  } = lineItems.edges[0].node.artwork
-  const truncateTextStyle = {
-    whiteSpace: "nowrap",
-    overflow: "hidden",
-    textOverflow: "ellipsis",
-  } as any
+export class TransactionSummary extends React.Component<
+  TransactionSummaryProps
+> {
+  formattedTaxTotal = () => {
```

Comment:
Is this being used? looks like you went with a `formattedAmount` helper (to work for tax and shipping)